go/src/lib.rs

```diff
@@ -70,6 +77,7 @@ pub enum ResponseType {
     String = 4,
     Array = 5,
     Map = 6,
+    Sets = 7,
```

Comment:
You can re-use array for that, no?

Reply:
@Yury-Fridlyand yes, we can do it. I've changed it.